helm/values.yaml

```diff
@@ -0,0 +1,106 @@
+#global values to be used across charts
+global:
+  namespace: default
+  serviceName: opendj
+  nginxIp: 192.168.99.100
+  oxAuthServiceName: oxauth
+  oxTrustSeriveName: oxtrust
+  domain: demoexample.gluu.org
+  gluuLdapUrl: opendj:1636
+  gluuMaxRamFraction: 1
+  configAdapterName: kubernetes
+  configSecretAdapter: kubernetes
+  #persistence types and options
+  # options: ldap/couchbase/hybrid
+  gluuPersistenceType: ldap
+  # options: default/user/site/cache/statistic; used only if GLUU_PERSISTENCE_TYPE is hybrid
+  gluuPersistenceLdapMapping: default
+  # used only if GLUU_PERSISTENCE_TYPE is ldap or hybrid
+  # used only if GLUU_PERSISTENCE_TYPE is couchbase or hybrid
+  gluuCouchBaseUrl: couchbase
+  #cloud provisioner. Will be used in wrends for dymanic provisioning
+  provisioner: k8s.io/minikube-hostpath
+
+# Required environment variables for generating Gluu server initial config
+# values for config-init sub-chart
+config:
+  enabled: true
+  orgName: Gluu
+  # email address of the administrator usually. Used for certificate creation
+  email: support@gluu.com
+  # adminPass is administrator password for oxTrust and LDAP
+  adminPass: P@ssw0rd
+  # domain name where the Gluu Server resides
+  domain: demoexample.gluu.org
+  # organisation location details
+  countryCode: US
+  state: TE
+  city: SLC
+  # ldapType only OpenDJ is supported
+  ldapType: opendj
+  
+  # <<<---------------------- SKIPPED THIS IN THE DOCUMENTATION -------->>>
+  # StorageClass values
+  # values used to define the StorageClass template
+  encrypted: true
+  reclaimPolicy: Retain
+  pool: default
+  description: gluu-config-init-volume
+  fsType: ext4
+  provisioner: k8s.io/minikube-hostpath
```

Comment:
add proviosner support for all the other images . Should include microk8s, GKE and AWS